src/main/java/org/dita/dost/writer/SeparateChunkTopicParser.java

```diff
@@ -310,6 +329,9 @@ public void startElement(final String uri, final String localName, final String
                     outputFile = generateOutputFilename(id);
                     output = new OutputStreamWriter(new FileOutputStream(new File(outputFile)), UTF8);
 
+                    if(atts.getIndex(ATTRIBUTE_NAME_XML_LANG) < 0 && !isEmptyString(currentLang)) {
```

Comment:
Don't use `isEmptyString`, it should be deprecated. Instead, just test for `null`